frappeclient/frappeclient.py

```diff
@@ -1,11 +1,11 @@
 import requests
 import json
-
+from urllib.parse import quote
```

Comment:
if we are doing this, we might as well only allow frappe client to be setup on python 3

Reply:
How about adding this block
`try:`
`	from urllib.parse import quote`
`except:`
`	from urllib import pathname2url as quote`

Reply:
there's quote in both python2 and python3

```python
try:
    from urllib import quote
except ImportError:
    from urllib.parse import quote
```

but I am wondering if we should completely drop support for python2 now.

Reply:
@Thunderbottom So have you decided which direction we are going ?

Reply:
@ahmadpak, we can drop python2 support from here

Reply:
Ok great I will do the suggest changes soon.